website/docs/dapp/_SwitchToTestnet.mdx

```diff
@@ -1,23 +1,9 @@
-So now your app works great on the local blockchain, you might want to switch it to different environments like Testnet and Mainnet.
+So now your dApp works great on the local blockchain, you might want to switch it to different environments like Testnet and Mainnet.
 
-To do this, you need to update the chain config and related code.
-
-Open the `ckb.ts` in your project root dir, change the `lumosConfig` and `CKB_RPC_URL`:
-
-```ts
-//export const lumosConfig: config.Config = devConfig as config.Config;
-export const lumosConfig = config.predefined.AGGRON4 as config.Config;
-
-//export const CKB_RPC_URL = 'http://localhost:8114';
-export const CKB_RPC_URL = "https://testnet.ckb.dev/rpc";
-```
-
-Actually, we have the corresponding Testnet version examples for all these tutorials. The source code of the Testnet version is in [https://github.com/nervosnetwork/docs.nervos.org/tree/develop/examples](https://github.com/nervosnetwork/docs.nervos.org/tree/develop/examples), you can clone the repo and start running on Testnet.
+To do that, just changes the environments variable `NETWORK`:
 
 ```sh
-git clone https://github.com/nervosnetwork/docs.nervos.org.git
-cd docs.nervos.org/examples/<example-name>
-yarn && yarn start
+NETWORK=testnet yarn start
```

Comment:
Should we move `yarn start` to a new line (or we can just delete that)

Reply:
in that case, it should be `export NETWORK=testnet`?